data/campaigns/Two_Brothers/scenarios/01_Rooting_Out_a_Mage.cfg

```diff
@@ -47,32 +47,40 @@
     [story]
         [part]
             music=revelation.ogg
-            story=_ "The remote freehold of Maghre in the western reaches of the Kingdom of Wesnoth was once a peaceful place, its inhabitants largely unaware of the comings and goings of the wider world. Wars and the rumor of wars touched them not, until the day a dark mage settled in the region and began seeking sacrifices for his evil summonings."
-            background=story/Two_Brothers_M1P1.webp
+            # po: This text is shown with the journey map, showing the village in the foothills along the route between Aldril and Dan’Tonk.
+            # po: I'm assuming the beacons either use smoke signals or basic fire signals, with a few agreed ones to request goods or indicate that surplus is for sale.
+            story=_ "The remote freehold of Maghre in the western reaches of the Kingdom of Wesnoth was once a peaceful place, its inhabitants largely unaware of the comings and goings of the wider world. Wars and the rumor of wars touched them not.
+
+Even merchants were rarely seen. Caravans between Aldril and Dan’Tonk would sometimes send a wagon to the village when called by the beacon tower."
+            {TB_BIGMAP}
+            {JOURNEY_REST_BEFORE_STAGE1}
         [/part]
         [part]
-            story=_ "Skeletons and zombies killed cattle and fired fields. <i>“Fear and obey Mordak the Mage!”</i> they cried in fell voices as they did their foul deeds. People vanished from isolated farmsteads. Men and women began to fear the night, and their children even the bright day. But the nearest lord was more than a day’s ride distant, and messengers sent to seek his help did not return."
+            story=_ "Then came the day that a dark mage settled in the region and began seeking sacrifices for his evil summonings.
```

Comment:
```suggestion
            story=_ "Then came the day that a dark mage settled in the region and began seeking sacrifices for his summonings.
```

Reply:
Taking "evil" out of this one makes it closer to having an alliteration.